code/modules/uplink/uplink_items.dm

```diff
@@ -142,6 +142,12 @@
 		spawned_item = spawn_path
 	if(refundable)
 		spawned_item.AddElement(/datum/element/uplink_reimburse, (refund_amount ? refund_amount : cost))
+
+
+	if(uplink_item_flags & SYNDIE_TRIPS_CONTRABAND) // Ignore things that shouldn't be detectable as contraband on the station.
+		ADD_TRAIT(spawned_item, TRAIT_CONTRABAND, INNATE_TRAIT)
+		for(var/obj/contained as anything in spawned_item.get_all_contents())
+			ADD_TRAIT(contained, TRAIT_CONTRABAND, INNATE_TRAIT)
```

Comment:
Side note, because it's *all* contents, this means stuff like space suit cells / modsuit mods will be marked as contraband, dunno if you care.

I guess you could technically metagame by scanning every discarded cell you find to see if any originated from an uplink but I doubt anyone would do that